js/utils.js

```diff
@@ -1512,62 +1512,30 @@ Fliplet.Registry.set('dynamicListUtils', (function() {
   }
 
   function setFilterValues(options) {
+    // Set options default in case it's undefined
```

Comment:
We don't need this comment. We do this a lot. It looks like you've included comments from my code examples. Those comments were to walk you through the code example. We don't need all of them.

```suggestion
```

Reply:
@tonytlwu,  removed the comments